lib/client-assets.php

```diff
@@ -1318,3 +1322,143 @@ function editor_color_palette_slugs( $color_palette ) {
 
 	return $new_color_palette;
 }
+
+define( 'GUTENBERG_POLYFILL_SCRIPT_HANDLE', 'wp-polyfills' );
+define( 'GUTENBERG_POLYFILL_FEATURE_TEST_PREFIX', 'wp-polyfill-test-' );
+
+/**
+ * Enqueues the root polyfill script.
+ */
+function gutenberg_enqueue_root_polyfill_script() {
+	wp_register_script( GUTENBERG_POLYFILL_SCRIPT_HANDLE, '' );
+	wp_enqueue_script( GUTENBERG_POLYFILL_SCRIPT_HANDLE );
+}
+add_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_root_polyfill_script', -9999 );
```

Comment:
🔨 (Excessive priority)

Reply:
heh 😄 Honestly, I'm pretty new to WP and am not sure what would be a reasonable priority here. My purpose was to make the idea clear that these things should always happen before printing scripts and, if included with WP 5.0, might be added as statements in the script loader rather than attached via hooks. To help me tone it down in the future, what priority would you use for something like this?

Reply:
It's the classic `z-index` issue: Everyone always thinks their use-case is the zenith.

The default priority is `10`. Usually if you need it to occur before the bulk of things, anything below that number should be sufficient (`5` or `9`, for example). I don't think a priority should ever be negative.